jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -45,44 +45,29 @@
       </v-expansion-panels>
     </v-row>
 
-    <!-- Composite region cannot be edited, so just grab first element. -->
-    <div v-if="is_editable">
-      <v-row v-for="(item, index2) in subset_definitions[0]"
+    <!-- Show all subregions of a subset, including Glue state and subset type. -->
+    <v-row v-for="(region, index) in subset_definitions"
+    class="pt-0 pb-0 mt-0 mb-0" align=center>
```

Comment:
what are all the `pt-0 pb-0 mt-0 mb-0` classes intended to accomplish?  Can these be simplified to get constant gutters (the inputs seem to extend all the way to the edge whereas the "subset type" and "glue state" rows do not) and perhaps use horizontal lines to delineate between entries?

Reply:
Was trying different approaches and this one seemed to get me the closest to what I wanted, happy to change it! Can you direct me to resources for using gutters in the way you're describing? Or somewhere else in our code?

Reply:
Maybe see https://jdaviz.readthedocs.io/en/latest/dev/ui_style_guide.html and feel free to update if you need something that isn't covered there?

Reply:
Removed the classes and have been trying all kinds of alignments but I cannot get the UI elements to stretch horizontally across the plugin even if the width is changed. I looked at similar plugin `.vue` files and I cannot see what I'm doing wrong. `guassian_smooth.vue` for example uses a `<v-text-field>` inside a `<v-row>` and that seems to be sufficient. Is it because I am using a `<v-row>` inside of another `<v-row>` that I am not seeing the same behavior?

Reply:
It's possible - can you avoid nesting rows?  It would be nice to keep everything to just individual rows (and even avoid columns, unless absolutely necessary).